src/main/java/ch/njol/skript/expressions/ExprLevelProgress.java

```diff
@@ -36,7 +36,7 @@
  * @author Peter Güttinger
  */
 @Name("Level Progress")
-@Description({"The progress a player has made until the next level. Remember that this value is between 0 and 1, not 0 and 100!",
+@Description({"The progress a player has made until they reach the next level. Remember that this value is between 0 and 1, not 0 and 100!",

```

Comment:
Well, not your fault since it's already been like that but I really find the whole first sentence weird lol.

Reply:
What about `The progress of a player to reach to the next level.`

Reply:
`The amount of experience the player needs before they reach the next level`?

Reply:
`The amount of experience the player needs to level up`

Reply:
Then what you've just committed is wrong...?

Reply:
lol

Reply:
wait what

Reply:
oh i just followed @Snow-Pyon's sentence since that made the most sense lol

Reply:
`The player's progress in reaching the next experience level, where 0 is an empty experience bar and 1 is the full bar.` maybe? Maybe some other word instead of `reaching`

Reply:
`where 0 is an empty experience bar and 1 is the full bar` looks weird and not clear to me

```
The player's progress in reaching the next level, this represents the experience bar in the game. Please note that this value is between 0 and 1 (e.g. 0.5 = half experience bar).
```

Reply:
Thanks @Blueyescat!